.circleci/jobs/features/catalogs.yml

```
@@ -0,0 +1,66 @@
+executor-machine: &executor-machine 'ubuntu-2004:202111-02'
```

```suggestion
executor-machine: &executor-machine 'ubuntu-2004:2022.04.1'
```
following https://github.com/akeneo/pim-enterprise-dev/pull/13669/files :)

---

good catch